azure-pipelines.yml

```diff
@@ -1,19 +1,42 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
 - master
 
-pool:
-  vmImage: 'ubuntu-latest'
+pool: UKHO Ubuntu 1804
+
+workspace:
+  clean: all
 
 steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+- task: CmdLine@2
+  inputs:
+    script: 'chmod 777 -R $(Build.ArtifactStagingDirectory)'
+  displayName: "Set chmod permissions (ArtifactStagingDirectory)"
+
+- task: CmdLine@2
+  inputs:
+    script: 'docker run --rm --mount type=bind,source=$(Build.ArtifactStagingDirectory),target=/zap/wrk/ -t owasp/zap2docker-stable zap-full-scan.py -t $(ApplicationUrl) -g gen.conf -r OWASP-Zap-Report.html -x Report.xml || true'
+  displayName: "Run Owasp Zap Full Scan"
```

Comment:
Minor - should we use the proper title "OWASP ZAP"

Reply:
Updated in lastest push